src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -111,42 +106,12 @@ private SscsCaseDetails createCaseInCcd(SscsCaseData caseData, IdamTokens idamTo
         }
     }
 
-    protected SscsCaseData transformAppealToCaseData(SyaCaseWrapper appeal) {
-        SscsCaseData caseData = submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal);
-
-        return updateCaseData(caseData);
-    }
-
-    protected SscsCaseData transformAppealToCaseData(SyaCaseWrapper appeal, String region, RegionalProcessingCenter rpc) {
-
-        SscsCaseData caseData = submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal, region, rpc);
-
-        return updateCaseData(caseData);
+    private SscsCaseData transformAppealToCaseData(SyaCaseWrapper appeal) {
+        return submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal);
```

Comment:
Could you make this method static? It does not hold any status and it does not need of any injection services, so in theory, it could be static and that will fix the sonar issue since we don't need to injected etc. Thanks